### PR TITLE
New Bluespace/Sepia slime reactions, new var for turfs

### PR DESCRIPTION
**Pull request by ghost**
Turfs now have a slowdown var that will determine how fast mobs move on
it.

Bluespace slimes can now create bluespace floor tiles which give -1
slowdown using blood

Sepia slimes can now create sepia floor tiles which give 2 slowdown
using blood

Camera and film reactions merged into a single water reaction for sepia

Bluespace crystal reaction moved to plasma

### CODE REVIEW

**Comment by PKPenguin321** [comment]
Pretty neat
Hopefully it will see more use than no-slip floor tiles

**Comment by KorPhaeron** [comment]
@PKPenguin321 

One would hope so. I think controlling speed is far less niche than making the tiles immune to the janitor

**Comment by xxalpha** [comment]
I like this.

**Comment by Lumipharon** [comment]
This is neat, but it would also prevent the return of the old superior bluespace plasma reaction (unless they current bluespace+blood reaction, because who the fuck even uses that?).

**Comment by KorPhaeron** [comment]
@Lumipharon

If I ever got it working/approved again I'd move the crystal thing to water (or remove it)

**Comment by Kromgar** [comment]
Do we have ZA WARUDO yet?

**Comment by KorPhaeron** [comment]
@Cheridan @RemieRichards 

Since you're the ones who always seem to look at my PRs I'd appreciate if someone reviewed the code when they got the chance. It's not very complicated so I'm hoping it's fine/pretty already

**Comment by RemieRichards** [comment]
Looks ok, floortiles/turfs in general with tiles are a bit finnicky so it'd be best to test that they actually place down and remove with a crowbar correctly.

**Comment by KorPhaeron** [comment]
@RemieRichards 

I did so (several times actually I'm deathly afraid of the embarrassment of submitting nonfunctional code)

File: code/modules/mob/mob_movement.dm [hunk 1]
```diff
@@ -137,8 +137,12 @@
 
 	if(isturf(mob.loc))
 
+
+		var/turf/T = mob.loc
 		move_delay = world.time//set move delay
 
+		move_delay += T.slowdown
```
Comment (phil235) [inline code/modules/mob/mob_movement.dm]: if mob.loc isn't a turf this will runtime.
You need var/turf/T = get_turf(mob) if(T) move_delay += T.slowdown

Comment (RemieRichards) [inline code/modules/mob/mob_movement.dm]: Incorrect @phil235 , I was going to report this myself, but it is below an ```if(isturf(mob.loc))``` check

Comment (phil235) [inline code/modules/mob/mob_movement.dm]: woops

**Pull request merged**